Cake.Recipe/Content/build.cake

```diff
@@ -280,7 +274,20 @@ public void CopyBuildOutput()
         // Now we need to test for whether this is a unit test project.  Currently, this is only testing for XUnit Projects.
         // It needs to be extended to include others, i.e. NUnit, MSTest, and VSTest
         // If this is found, move the output to the unit test folder, otherwise, simply copy to normal output folder
-        foreach(var reference in parsedProject.References)
+
+        ICollection<ProjectAssemblyReference> references = null;
+        if(!BuildParameters.IsDotNetCoreBuild)
```

Comment:
not sure how this is wired up but parsedProject from 1.2.0 onwards contains the following bools to allow directing compilation tasks as appropriate
`parsedProject.IsNetFramework`  // <= VS2015
`parsedProject.IsNetCore`  // <= VS2017
`parsedProject.NetCore.IsWeb`

Just thinking the names might be misleading though as you would still need to check the values in  `parsedProject.NetCore.TargetFrameworks` for the right compilation